challenge_gpsr/src/gpsr.py

```diff
@@ -74,7 +75,7 @@ def task_finished(self, message):
         else:
             rospy.logwarn("Not going to meeting point, challenge has param skip:=true set")
 
-        if self.tasks_done >= self.tasks_to_be_done and not self.skip:
+        if (self.tasks_done >= self.tasks_to_be_done and not self.skip) or (self.finished and not self.skip and self.tasks_done >= 1):
```

Comment:
This becomes very complex...

Reply:
I simplified this a little bit, however this addition ensures that the robot drives out of the arena when there is only a little bit of time left. This should result in a very high chance to obtain extra points for autonomously driving out of the arena while just doing an extra assignment has a lower chance of winning more points.

Reply:
IMO it would be best to only have a finished check here. You can keep the skip here as well. But finished should be determined somewhere else. That logic can be more complex.

Reply:
Let's meet half way:

```
 if (self.tasks_done >= self.tasks_to_be_done or self.finished) and not self.skip:
```